go/ubirch/ubirch_sim.go

```diff
@@ -75,11 +75,11 @@ func (p *Protocol) encodeBinary(tags []Tag) []byte {
 		if length <= 0xff {
 			encoded = append(encoded, byte(length))
 		} else {
```

Comment:
What happens if length is > 65535 (0xffff)? Shouldn't we throw an error or a panic?

Reply:
added todo